server/test/adminRoutes.spec.js

```diff
@@ -1,121 +1,107 @@
 import chai from 'chai';
-
- import chaiHttp from 'chai-http';
+import { describe, it } from 'mocha';
+import chaiHttp from 'chai-http';
 
 import app from '../../app';
 
-const expect = chai.expect;
+const Expect = chai.expect;
 
 chai.use(chaiHttp);
 
 
-describe('ADMIN ROUTES',() =>{
-    //Get all orders
-it('Should Get All orders',(done)=>{
+describe('ADMIN ROUTES', () => {
+  // Get all orders
+  it('Should Get All orders', (done) => {
     chai.request(app)
-    .get("/api/v1/allOrders")
-    .then((res) =>{
-        expect(res).to.have.status(200);
-        expect(res).to.be.an('array')
-    })
+      .get('/api/v1/allOrders')
+      .then((res) => {
+        Expect(res).to.have.status(200);
```

Comment:
Try to assert that the array of orders is actually returned> Let me know if you need clarification on this